source/GameGraphRoot.cpp

```diff
@@ -747,9 +747,9 @@ void GameGraphRoot::update(float timestep) {
 	// Rotate nearSpace about center.
 	float newPlayerAngle = ship->getDonuts().at(playerID)->getAngle();
 	float delta = (prevPlayerAngle - newPlayerAngle) * globals::PI_180;
-	delta = delta < -globals::PI
-				? delta + ship->getSize() * globals::PI_180
-				: delta > globals::PI ? delta - ship->getSize() * globals::PI_180 : delta;
+	delta = (delta < -globals::PI)
+				? (delta + ship->getSize() * globals::PI_180)
+				: (delta > globals::PI ? delta - ship->getSize() * globals::PI_180 : delta);
```

Comment:
This was just to make `clang-format` happy; I suspect that our CI pipeline is running a newer version of `clang-format` than Visual Studio is, because it didn't show up as a problem in VS. Nonetheless, adding this fixed it.

Reply:
Yep; the CI is using `llvm 11` while VS is on `llvm 10`. I'll see if there's anything I can do about that.